www/index.html

```diff
@@ -36,7 +36,7 @@
     then follow the onscreen instructions.
   </p>
   <p>If you're a Windows Subsystem for Linux user run the following in your terminal, then follow the onscreen instructions to install Rust.</p>
-  <pre>curl https://sh.rustup.rs -sSf | sh</pre>
+  <pre>curl --proto =https --tlsv1.2 -sSf https://sh.rustup.rs | sh</pre>
```

Comment:
Could this be left as it was? This is just intended to be a quick one-liner copy/paste snippet and expanding with so many options can be pretty intimidating as a copy/paste snippet. Does this really add enough benefit relative to what it's displaying on the site?

Reply:
You are expressing certain feelings towards this without backing them up with a UX study. I am sad you prioritize the length of an already technical one-liner over the clear and specified security improvement. I think it is instead a good signal that in Rust ecosystem, security is taken seriously.

Reply:
Certainly the display isn't as good as it used to be.  We *could* use web techniques to hide the extra arguments from the display but include them in the copy/paste, and provide a copy-command-to-clipboard button which puts it all there, for ease of use?

Reply:
Why would that be important?

Reply:
Currently the box requires scrolling, and people are more likely to fail to copy the whole command than not.  While I personally dislike the narrow styling, I understand why it's there so would prefer to maintain it over trying to do better myself.  If someone else could make it work somehow then that'd be good too.